src/App.vue

```diff
@@ -17,24 +16,16 @@ const isAuthLoaded = ref(false)
 onBeforeMount(async () => {
   store.commit('showLoader', 'Initializing Arcana Auth SDK...')
   await arcanaAuth.init()
-  store.commit('showLoader', 'Fetching app configuration...')
-  const encodedAccessToken = localStorage.getItem('access-token')
-  const userInfo = localStorage.getItem('user-info')
-  if (encodedAccessToken && userInfo) {
-    const accessToken = new TextDecoder().decode(
-      cryptoUtils.stringToBuffer(encodedAccessToken)
-    )
-    store.dispatch('updateAccessToken', accessToken)
```

Comment:
Question: Where else do we save the access token? I believe this is required for the gateway and smart contract calls.

Reply:
Yeah that we get from login, we don't need to store it in localstorage